### PR TITLE
Improve RTD and README

### PR DESCRIPTION
**Pull request by msmdev**
* Add Acknowledgements to README.
* Add Acknowledgements section to index.rst.
* Make some style improvements.
* Add Licence and Download badges.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,44 @@
...
+    :target: https://pygpcca.readthedocs.io/en/latest
+    :alt: Documentation
+
+.. |Coverage| image:: https://img.shields.io/codecov/c/github/msmdev/pygpcca/main
+    :target: https://codecov.io/gh/msmdev/pygpcca
+    :alt: Coverage
+
+.. |License| image:: https://img.shields.io/github/license/msmdev/pyGPCCA?color=green
+    :target: https://github.com/msmdev/pyGPCCA/blob/main/LICENSE.txt
+    :alt: GitHub
```
Comment (michalk8) [inline README.rst:29]: Alt. should be License.

File: README.rst [hunk 2]
```diff
@@ -1,4 +1,44 @@
...
+
+.. |GitHubDownloads| image:: https://img.shields.io/github/downloads/msmdev/pyGPCCA/total?label=github%20downloads
```
Comment (michalk8) [inline README.rst:31]: Not sure if this is necessary.

File: README.rst [hunk 3]
```diff
@@ -1,4 +1,44 @@
...
+    :target: https://github.com/msmdev/pyGPCCA/releases/
+    :alt: GitHub all releases
+
+.. |PyPIdownloads| image:: https://img.shields.io/pypi/dm/gpcca?label=pypi%20downloads
```
Comment (michalk8) [inline README.rst:35]: This is already done above, see |PyPI|. If you want the number of downloads from PyPI, you can use: http://pepy.tech/

File: README.rst [hunk 4]
```diff
@@ -1,4 +1,44 @@
...
+    :target: https://pypi.org/project/pygpcca/
+    :alt: PyPI - Downloads
+
+.. |CondaDownloads| image:: https://img.shields.io/conda/dn/conda-forge/pygpcca?label=conda%20downloads
```
Comment (michalk8) [inline README.rst:39]: Same as for PyPI - already done with |Conda| alias.

File: README.rst [hunk 5]
```diff
@@ -42,32 +88,28 @@ Example
...
-    :alt: Coverage
-
-.. _`PETSc`: https://www.mcs.anl.gov/petsc/
+Acknowledgements
+----------------
+We thank `Marcus Weber`_ and the Computational Molecular Design (`CMD`_) group at the Zuse Institute Berlin (`ZIB`_)
+for the longstanding and productive collaboration in the field of Markov modeling of non-reversible molecular dynamics.
+M. Weber, together with K. Fackeldey, had the original idea to employ Schur vectors instead of eigenvectors in the
+coarse-graining of non-reversible transition matrices years ago. |br|
+Further, we would like to thank `Fabian Paul`_ for valueable discussions regarding the sorting of Schur vectors and his
```
Comment (michalk8) [inline README.rst:97]: Typo: valueable - should be valuable.

File: docs/source/acknowledgements.rst [hunk 6]
```diff
@@ -0,0 +1,22 @@
+Acknowledgements
+================
+We thank `Marcus Weber`_ and the Computational Molecular Design (`CMD`_) group at the Zuse Institute Berlin (`ZIB`_)
+for the longstanding and productive collaboration in the field of Markov modeling of non-reversible molecular dynamics.
+M. Weber, together with K. Fackeldey, had the original idea to employ Schur vectors instead of eigenvectors in the
+coarse-graining of non-reversible transition matrices years ago.
+Further, we would like to thank `Fabian Paul`_ for valueable discussions regarding the sorting of Schur vectors and his
```
Comment (michalk8) [inline docs/source/acknowledgements.rst:7]: Same type - valueable should be valuable.

File: docs/source/index.rst [hunk 7]
```diff
@@ -25,32 +11,9 @@ under study. Utilizing *pyGPCCA*, metastable states as well as cyclic kinetics c
...
-
-.. |Docs|  image:: https://img.shields.io/readthedocs/pygpcca
-    :target: https://pygpcca.readthedocs.io/en/latest
-    :alt: Documentation
-
-.. |Coverage| image:: https://img.shields.io/codecov/c/github/msmdev/pygpcca/main
-    :target: https://codecov.io/gh/msmdev/pygpcca
-    :alt: Coverage
-
 .. |br| raw:: html
```
Comment (michalk8) [inline docs/source/index.rst:17]: If |br| is no longer used, it can be removed.

**Review by michalk8 — CHANGES_REQUESTED** [review]

File: README.rst (hunk 4 above)
Comment (msmdev) [reply README.rst:39]: @michalk8 I thought that |Conda| links the version <https://img.shields.io/pypi/v/pygpcca> while |CondaDownloads| would link to the # of downloads <https://img.shields.io/conda/dn/conda-forge/pygpcca?label=conda%20downloads>?

Comment (michalk8) [reply README.rst:39]: The `img.shields.io` is where we are getting the badge, it's the `:target:` what matters (i.e. when user click the badge, they go to the target ).

File: README.rst (hunk 1 above)
Comment (msmdev) [reply README.rst]: Done.

File: README.rst (hunk 2 above)
Comment (msmdev) [reply README.rst]: Removed.

File: README.rst (hunk 3 above)
Comment (msmdev) [reply README.rst]: Done.

File: README.rst (hunk 4 above)
Comment (msmdev) [reply README.rst]: Alright, removed :)

File: README.rst (hunk 5 above)
Comment (msmdev) [reply README.rst]: Done.

File: docs/source/acknowledgements.rst (hunk 6 above)
Comment (msmdev) [reply docs/source/acknowledgements.rst]: Done.

File: docs/source/index.rst (hunk 7 above)
Comment (msmdev) [reply docs/source/index.rst]: Done.

**Comment by msmdev** [comment]
@michalk8 guess I covered everything? Can I merge?

File: docs/source/index.rst (hunk 7 above)
Comment (msmdev) [reply docs/source/index.rst:17]: Was indeed needed, since the beginning of the readme uses |br| and is included in index.rst.

**Comment by michalk8** [comment]
@msmdev looks good, please go ahead and merge.

**Comment by michalk8** [comment]
Wait - I think the `:target:` in `pepy.tech` is wrong - it shows `requests`.
And there' some `Conda/Installation` in the `index.rst` in the docs, left of the contributors.

**Comment by msmdev** [comment]
Ooops, didn't see your last comment...
Will tend to that in a new PR...

**Pull request merged**